### DISCUSSION TITLE
Generate uuid / identifier

### DISCUSSION THREAD

**Discussion by darthf1** [duckdb/duckdb#2341] [Q&A] [answered]
Hi!

I'm currently using DuckDB (python) to perform SQL queries over multiple parquet files. It works great!

I'm looking for a way to generate an unique id per row, like the `gen_random_uuid()` function in [PostgreSql](https://www.postgresql.org/docs/13/functions-uuid.html).

I could not find anything in the documentation related to this functionality. Is it possible to do something as such at the moment?

**Comment by Mytherin** [comment, marked as answer]
UUIDs are not supported yet. Feel free to open an issue with a feature request!
